firestore/query.go

```diff
@@ -61,6 +62,149 @@ type Query struct {
 	// readOptions specifies constraints for reading results from the query
 	// e.g. read time
 	readSettings *readSettings
+
```

Comment:
Could you add a comment explaining the purpose of `runQuerySettings` for consistency?

Reply:
Added